var/spack/repos/builtin/packages/berkeleygw/package.py

```diff
@@ -147,19 +153,34 @@ def edit(self, spec, prefix):
         if self.version >= Version("3.0"):
             si_epm_tests.append("Si_hdf5")
         for test in si_epm_tests:
+            filter_file(
+                "Precision : 5e-12",
+                "Precision : 6e-12",
+                join_path("testsuite", "Si-EPM", test + ".test"),
+            )
             filter_file(
                 "Precision : 6e-15",
                 "Precision : 7e-15",
                 join_path("testsuite", "Si-EPM", test + ".test"),
             )
-        for test in ["Si_subspace", "Si_subspace_cplx", "Si_subspace_cplx_spin"]:
+
+        si_epm_subspace_tests = ["Si_subspace", "Si_subspace_cplx_spin"]
+        if self.version < Version("4.0"):
+            si_epm_subspace_tests.append("Si_subspace_cplx")
+        for test in si_epm_subspace_tests:
             filter_file(
                 "Precision : 6e-15",
                 "Precision : 7e-15",
                 join_path("testsuite", "Si-EPM_subspace", test + ".test"),
             )
         filter_file("Precision : 8e-15", "Precision : 9e-15", "testsuite/GaAs-EPM/GaAs.test")
 
+        if self.version < Version("3.1.0"):
+            # np.int alias was removed from numpy
```

Comment:
(question) based on this comment, should this be based on the version of `py-numpy` that is used as well?

Reply:
it is true that the `numpy.int` alias would still work for `numpy` < 1.24, but the `int` builtin always works, so this seems simpler?

(https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations, https://numpy.org/devdocs/release/1.24.0-notes.html#expired-deprecations)

Reply:
That sounds good - thanks for the response!